cli/flox/doc/flox.md

```diff
@@ -98,32 +98,6 @@ sharing environments, and administration.
 
 # ENVIRONMENT VARIABLES
 
-`$FLOX_PROMPT_ENVIRONMENTS`
```

Comment:
I think it's fine to have them in the man page.
Instead of having copies in multiple manpages we can however move the section into an included file.

Reply:
I have a slight preference for only having them in one place, are you wanting me to factor it out or just think it's fine to have them in both?

Reply:
Fine by me. I think there are some variables that a really only relevant for 1-2 commands, so they don't have to be in a common page, but if we want them to be it doesn't necessarily hurt

Reply:
I'd prefer to have the environment variables relevant to a particular command in the docs for that command. The rationale is that if you're looking for information about the behavior for a command, you're going to look at the docs for that command. I also think it's less of a hassle for the user since they don't need to jump through multiple man pages to get the info they're looking for.

I'm not sure if having an included page makes sense because then you also get information about environment variables that aren't relevant to this command.

Reply:
In the end it's messier to have multiple potentially overlapping sets of environment variables in the multiple man pages, but I don't think we have _that_ many environment variables yet that we need to start coming up a more rigorous system. I think the large majority of the environment variables simply belong in the `activate` man page.